_translator-files/po/de/Include-Client-Commands.po

```diff
@@ -3,50 +3,51 @@
 # This file is distributed under the same license as the PACKAGE package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
 #
-#, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
+"Project-Id-Version: \n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: 2022-05-16 17:41+0200\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: de\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 3.0.1\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Client-Commands.md:2
 #, no-wrap
 msgid "`-M` or `--mutestream`  Starts Jamulus in muted state                                                      \n"
-msgstr ""
+msgstr "`-M` or `—mutestream` Startet Jamulus im stummen Zustand\n"
```

Comment:
I think it's worth changing the documentation depending on the outcome of https://github.com/jamulussoftware/jamulus/issues/2335 - Although it's unrelated to this PR.